docs/index.yml

```diff
@@ -251,6 +251,15 @@ additionalContent:
       - title: ".NET for Apache Spark API reference"
         summary: API reference documentation for .NET for Apache Spark
         url: https://docs.microsoft.com/dotnet/api/?view=spark-dotnet
+      - title: ".NET Core"
```

Comment:
I don't think this is the right spot for these links, because it's under API reference. We already have links to .NET Core and .NET Standard in the first two cards. Maybe just move ASP.NET Core to the second card ".NET concepts"? Also, the feedback mentioned having a link to EF Core.

Reply:
Great suggestion!